lib/readline.js

```diff
@@ -47,9 +47,9 @@ function Interface(input, output, completer, terminal) {
   }
   historySize = historySize || kHistorySize;
 
-  completer = completer || function() { return []; };
+  completer = completer || undefined;
```

Comment:
Works for me. I guess it's a bit of embracing being somewhat liberal in what you accept (any falsy value as opposed to specifically `undefined`). Unless tests unexpectedly fail with the change, I'll commit that and the doc change in a little bit...